project3/styles.css

```
@@ -0,0 +1,373 @@
+* {
+    box-sizing: border-box;
+    transition: all 375ms linear;
+}
+
+.container {
+    background-color: hsl(228, 48%, 84%);
+    margin: 0 auto;
+    width: 375px;
+}
+
+header {
+    padding-top: 1px;
+    padding-bottom: 1px;
+    background-color: #d1d5de;
+}
+
+
+
+
+
+.logo {
+    height: 239px;
+    width: 240px;
+    margin-left: auto;
+    margin-right: auto;
+    margin-top: 40px;
+    margin-bottom: 20px;
+    display: block;
+    position: relative;
+}
+
+.identity {
+    text-align: left;
+    margin-top: 20px;
+}
+
+.personal {
+    margin-top: 60px;
+    text-align: left;
+}
+
+.age {
+    margin-top: 38px;
+    margin-left: 16px;
+}
+
+.category {
+    margin-top: 30px;
+    margin-bottom: 18px;
+    margin-left: 16px;
+}
+
+.field-margin {
+    margin-top: 30px;
+    margin-bottom: 18px;
+    margin-left: 16px;
+}
+
+.margins {
+    margin-left: 18px;
+    margin-right: 18px;
+}
+
+#powers {
+    margin-left: 18px;
+    margin-right: 18px;
+    height: 68px;
+    width: 291px;
+}
+
+.bulb {
+    height: 100px;
+    width: 100px;
+    margin-top: 40px;
+}
+
+
+
+.bulb {
+    height: 100px;
+    width: 100px;
+    margin-top: 40px;
+}
+
+
+.muscle {
+    height: 101px;
+    width: 100px;
+}
+
+.watch {
+    height: 100px;
+    width: 100px;
+    margin-top: 40px;
+}
+
+.caption {
+    margin-top: 19px;
+    margin-left: 150px;
+    margin-right: 50px;
+}
+
+.text-box {
+    height: 73px;
+    width: 240px;
+    margin-left: 50px;
+    margin-right: 50px;
+    margin-top: 40px;
+    margin-bottom: 40px;
+}
+
+svg {
+    display: block;
+    margin-right: 50px;
+    margin-left: 120px;
+    width: 100%;
+
+}
+
+
+
+:root {
+    --color-red: #c61a3a;
+    --color-green: #71f79f;
+    --color-offblack: #1e2019;
+    --color-grey: #d1d5de;
+    --color-offwhite: #FFF;
+    --color-blue: #163346;
+
+}
+
+*,
+*::before *::after {
```

Missing comma to separate selectors:
```suggestion
*::before, *::after {
```